src/components/mobile/mobileAssetView.tsx

```diff
@@ -0,0 +1,302 @@
+      return;
+    }
+  }
+
+  render() {
+    console.log(this.props.asset);
+    return (
+      <div className={Classes.DARK}>
+        <Callout title={"Asset Properties"}>
+          <table className="bp3-html-table">
+            {this.renderData(Object.keys(AssetFieldsTable), this.props.asset)}
+          </table>
+        </Callout>
+        <Callout title={"Model Properties"}>
+          <table className="bp3-html-table">
+            {this.renderData(
+              Object.keys(ModelFieldsTable),
+              this.props.asset.model
+            )}
+          </table>
+        </Callout>
+        <Callout title={"Network Connections"}>
+          {this.props.asset.model &&
+          this.props.asset.model.network_ports &&
+          this.props.asset.model.network_ports.length !== 0 ? (
+            <div className="network-connections">
+              <table className="bp3-html-table bp3-html-table-bordered bp3-html-table-striped">
+                <tr>
+                  <th>Network Port</th>
+                  <th>Mac Address</th>
+                  <th>Destination Asset</th>
+                  <th>Destination Port</th>
+                </tr>
+                <tbody>
+                  {this.props.asset.model.network_ports.map((port: string) => {
+                    var connection = this.getNetworkConnectionForPort(port);
+                    return (
+                      <tr>
+                        {" "}
+                        <td style={getChangePlanRowStyle(this.props.asset)}>
+                          {port}
+                        </td>
+                        <td style={getChangePlanRowStyle(this.props.asset)}>
+                          {this.props.asset.mac_addresses
+                            ? this.props.asset.mac_addresses[port]
+                            : null}
+                        </td>{" "}
+                        {connection
+                          ? [
+                              <td
+                                style={getChangePlanRowStyle(this.props.asset)}
+                                className={
+                                  this.props.asset.decommissioning_user
+                                    ? undefined
+                                    : "asset-link"
+                                }
+                                // onClick={
+                                //   this.props.asset.decommissioning_user
+                                //     ? undefined
+                                //     : (e: any) => {
+                                //         const id = this.getAssetIdFromHostname(
+                                //           connection!.destination_hostname!
+                                //         );
+                                //         if (id) {
+                                //           this.redirectToAsset(id);
+                                //         }
+                                //       }
+                                // }
```

Comment:
This is worth discussing. Do we want to be able to click to other assets in mobile? My gut tells me it probably doesn't matter one way or another, since the sense I got from the requirements was just that the mobileAssetView only needs to display "pertinent information" about the asset and does not specify any need for this functionality.

Reply:
if that's what the requirements say then I would lean towards no, I think this asset view can be simpler than the other one. It's probably worth clarifying this on piazza though?